spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'GET/show' do
+  context 'with success response' do
+    it 'should return success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Here is a list of posts with comments for a given user')
+    end
+
+    it 'should return success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Post details for a given user')
+    end
+  end
+end
```

Comment:
- Good job done so far, but currently all your tests are not passing, a possible reason for this is that you are not using the right request type. Your test file should look like this:
```ruby
   require 'rails_helper'

RSpec.describe 'Posts', type: :request do
  describe 'GET/show' do
    context 'with success response' do
      it 'should return success' do
        get '/users/'
        expect(response).to have_http_status(:success)
        expect(response.body).to include('Lists of all users')
      end
  
      it 'should return success' do
        get '/users/:id'
        expect(response).to have_http_status(:success)
        expect(response.body).to include('Information about a given user')
      end
    end
  end
end
```
- Please fix the above issue for both test files 👍 👍 
![srgasrga](https://github.com/Luchy-B/blogapp/assets/73607512/138e7915-daca-473d-8c63-01b68396ae52)